packages/gatsby-plugin-gatsby-cloud/src/components/Indicator.js

```diff
@@ -71,7 +142,8 @@ const Indicator = () => {
     }
 
     if (currentBuild?.buildStatus === BuildStatus.BUILDING) {
-      setBuildInfo({ ...newBuildInfo, buildStatus: BuildStatus.BUILDING })
+      // setBuildInfo({ ...newBuildInfo, buildStatus: BuildStatus.BUILDING })
+      setBuildInfo({ ...newBuildInfo, buildStatus: BuildStatus.UPTODATE })
```

Comment:
is there no more BUILDING state in the indicator?

Reply:
not for preview 2.0. We do not want to show the BUILDINg indicator or spinner until we are sure the build is applicable for this content, which is not possible until the build is successful and page data has updated for a non CS build.

Reply:
👍  good call. I haven't put too much thought into it since we will be updating this code for eager redirects, so we will modify this code in the next pr

Reply:
Decided to keep the way it is, as we dont want to modify the messaging on the building state, just have an extra check there to show updated state if build is not applicable.

Reply:
Sounds good! I'm a little confused as to what this comment means `Keep status as up to date for non content sync builds`. What's a non-content sync build?

Reply:
Modified it. If we ca't fetch manifest information, we cant tell if the build is applicable for this content.